test/e2e_new/data/cni/calico.yaml

```diff
@@ -0,0 +1,792 @@
+---
```

Comment:
How are we currently handling this in capi? I would like to see if we can avoid having a copied version of the calico manifest here if at all possible.

Reply:
CAPI is also using a duplicate copy. Using URLs would be cleaner. Created [CAPI 3117](https://github.com/kubernetes-sigs/cluster-api/issues/3117)